### DISCUSSION TITLE
New AI Friend

### DISCUSSION THREAD

**Discussion by Middelman** [Cyfrin/foundry-full-course-cu#159] [General]
Polygon has released thier AI chatbot called [Polygon Copilot](https://copilot.polygon.technology/) 
Another AI friend.

[Copilot User Guide](https://layer-e.gitbook.io/polygon-copilot-docs/)

**Comment by ChefAharoni** [comment]
That's awesome! Thank you for sharing.

**Comment by cromewar** [comment]
Thanks for sharing @Middelman
